kubernetes/TAG

```diff
@@ -1 +1 @@
-1.18.9.1
+1.18.9.2
```

Comment:
If you want to update this, update CKE too.
Personally, I don't think this needs to be updated now.

Reply:
Just to be clear, this change is intended to update its ubuntu base image. Do we need to update it when we upgrade k8s to 1.19?

Reply:
I know.

> Do we need to update it when we upgrade k8s to 1.19?

I don't understand your question.  This *is* the Kubernetes.

Reply:
Sorry, I don't understand your answer as well.

Reply:
To upgrade to Kubernetes 1.19, this _must_ be updated because this _is_ the Kubernetes.